rules_go_gapic/go_gapic_pkg.bzl

```diff
@@ -49,10 +49,6 @@ def _go_gapic_src_pkg_impl(ctx):
             mkdir -p {package_dir_path}/${{src_str%:*}}
             cp -f ${{src_str#*:}} {package_dir_path}/${{src_str%:*}}
         fi
-        # dest_dir_path is split again, on the ';' delimiter, for the
-        # Go importpath that includes the package name following ';'
-        s=${{src_str%:*}}
-        chmod 644 {package_dir_path}/${{s%;*}}/*
```

Comment:
I don't really know why this was needed in the first place, but it works without it, and actually removing it is necessary because for some reason the `snippets` dir becomes unreadable after this is run...